website/components/ecosystem-integration-group/ecosystem-card/index.jsx

```diff
@@ -0,0 +1,31 @@
+import s from './style.module.css'
+import Image from '@hashicorp/react-image'
+import SmartLink from 'components/smart-link'
+
+export default function EcosystemCard({
+  companyName,
+  integrationType,
+  integrationUrl,
```

Comment:
nit: `url` is shorter. (if you have thought of a reason why prefixing with `integration` is good, lmk!!)

Reply:
@mwickett Good points on keeping things shorter! My reasoning was I thought it might be better to be explicit about prop names for authors' sake.  Having it spelled out like this makes it easier to immediately see what information is expected for each card.